packages/tailwind/src/kz-spacing.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { filterOutNonPixelTokens } from "./kz-spacing"
+
+const numericKeyPairs = {
+  0: "0",
+  1: ".0625rem",
+  2: ".125rem",
+}
+const nonNumericKeyPairs = {
+  xs: "0.375rem",
+  sm: "0.75rem",
+}
+
+const dummyData = {
+  ...numericKeyPairs,
+  ...nonNumericKeyPairs,
+}
+
+describe("filterOutNonPixelTokens", () => {
+  const result = filterOutNonPixelTokens(dummyData)
+
+  test("It removes pairs with non-numeric keys", () => {
```

Comment:
For future reference, the way you're writing the test, you could use the `it()` syntax here:
```suggestion
  it("removes pairs with non-numeric keys", () => {
```

Reply:
Changed to "it" now